userspace/libsinsp/sinsp_filtercheck_thread.cpp

```diff
@@ -1107,10 +1126,16 @@ uint8_t* sinsp_filter_check_thread::extract_single(sinsp_evt* evt,
 		uint32_t j;
 		uint32_t nargs = (uint32_t)tinfo->m_args.size();
 
-		for(j = 0; j < nargs; j++) {
-			m_tstr += tinfo->m_args[j];
-			if(j < nargs - 1) {
-				m_tstr += ' ';
+		if(m_argid >= 0) {
```

Comment:
@FedeDP Suggesting to further clean this up after rebasing the next PR, new helper is introduced here https://github.com/falcosecurity/libs/pull/2387/files#r2074420790